xrpl/asyncio/clients/websocket_base.py

```diff
@@ -135,9 +136,10 @@ async def _handler(self: WebsocketBase) -> None:
             # if this response corresponds to request, fulfill the Future
             if "id" in response_dict and response_dict["id"] in self._open_requests:
                 self._open_requests[response_dict["id"]].set_result(response_dict)
-
-            # enqueue the response for the message queue
-            cast(_MESSAGES_TYPE, self._messages).put_nowait(response_dict)
+                # a response that fulfills a future is not enqueued again
```

Comment:
This comment is more confusing than useful IMO.

Reply:
fixed in [3a53ffc](https://github.com/XRPLF/xrpl-py/pull/713/commits/3a53ffc66ab7b99675d8a5fbb59aa52ae73d6d4f)